benchexec/tools/verioover.py

```diff
@@ -0,0 +1,50 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+# SPDX-FileCopyrightText: 2016-2020 Marek Chalupa
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    VeriOover
+
+    http://github.com/PaperSheeper/VeriOover-SV
+    """
+
+    def name(self):
+        return "VeriOover"
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("VeriOover")
+
+    def version(self, executable):
+        return self._version_from_tool(executable)
+
+    def cmdline(self, executable, options, task, rlimits):
+        if task.single_input_file:
```

Comment:
This check is not necessary. `single_input_file` will automatically throw if there is not exactly one input file, and BenchExec will provide a nice error message to the user. And the returned string would never be empty anyway. So just add it to `options` unconditionally.